transformations.html

```diff
@@ -754,9 +827,10 @@ <h2>String interpolation</h2>
   |]
  </code></pre>
 
-<pre><code class="roc haskell">
-costOnSale money =
+<pre><code class="roc coffeescript">
+costOnSale = \money ->
   euros = toEuros money
+  
   "It cost \(euros) when on sale."
```

Comment:
do we want to add a comment about let..in and local assignment?
```suggestion
costOnSale = \money ->
  euros = toEuros money -- there's no let..in block in Roc
  
  "It cost \(euros) when on sale."
```

also, I assume the sole example shown means the following is not valid?
```
costOnSale = \money ->
    "It cost \(toEuros money) when on sale."
```

Reply:
That's correct - string interpolation has to be just a variable name.

Could add a comment, although the `let` section already explains this. 🤷

Reply:
Ah my bad missed the Let section 👌